src/models/constraint.rs

```diff
@@ -72,7 +72,7 @@ impl Constraint {
 ///
 /// Usage:
 ///
-/// ```ignore
+/// ```
```

Comment:
What's this change doing? No longer ignoring these blocks for formatting?

Reply:
This `ignore` implies that cargo should ignore docstring tests. 
Now i specify that explicitly in the `Cargo.toml` file